xtask/src/publish.rs

```diff
@@ -11,7 +11,7 @@ const CARGO: &str = crate::CARGO;
 
 // in order for publishing
 const PACKAGES: &[&str] = &[
-    "crux_cli",
+    // "crux_cli",
```

Comment:
Intentional?

Reply:
kinda. It's better out for now, while we think about how to effectively treat the cli and the "rest" as 2 separate things